scripts/_helpers.py

```diff
@@ -10,10 +10,16 @@
 
 import country_converter as coco
 import geopandas as gpd
-import numpy as np
 import pandas as pd
+import yaml
 
 REGION_COLS = ["geometry", "name", "x", "y", "country"]
+with open("configs/osm_config.yaml", "r") as f:
```

Comment:
This path formulation seems to lead to issues on windows.

It may be nice to have an helper function that performs the following reading.
Potentially, the function may be something like:

```
read_osm_config(args*):
   with open("configs/osm_config.yaml", "r") as f:
      osm_config = yaml.safe_load(f)
  return tuple([osm_config[a] for a in args])
```
and then define world_iso, ... , osm_clean_columns = read_osm_config("world_iso", ...)

Using that definition, in the future it may be nice to revise world_iso as it may not be a global costant value, but only loaded when it makes sense.
Now, whenever the _helpers are loaded it would be loaded as well, which may not be necessary

Reply:
The definition of that function would enable to flexibly load only world_iso or any other object in that yaml file on demand, in the scripts that actually need them. Ad-hoc function may be created specifically for that too

Reply:
For this PR they can be global constants for this script anyway ;) easier and practical

Reply:
I did something like this initially and figured it was adding too many lines of code. Does make sense though. Let me test using `os.path.join` maybe this fixes the issue in windows.